basics/indexing/forward-index.md

```diff
@@ -1,15 +1,29 @@
 # Forward Index
 
-The values for every column are stored in a forward index, of which there are three types:
+At conceptual level, the forward index is a map from document id (aka row index) to the actual column value of the row.
+How this map is implemented depends on the index encoding and on whether the column is sorted or not.
 
-* [Dictionary encoded forward index](forward-index.md#dictionary-encoded-forward-index-with-bit-compression-default)\
-  Builds a dictionary mapping 0 indexed ids to each unique value in a column and a forward index that contains the bit-compressed ids.
-* [Sorted forward index](forward-index.md#sorted-forward-index-with-run-length-encoding)\
-  Builds a dictionary mapping from each unique value to a pair of start and end document id and a forward index on top of the dictionary encoding.
-* [Raw value forward index](forward-index.md#raw-value-forward-index)\
-  Builds a forward index of the column's values.
+When encoding is `RAW`, the forward index is implemented as an array whose indexes are the document ids and the values
+are the actual row value. 
+Read the [raw value forward type](forward-index.md#raw-value-forward-index) to learn more about that.
 
-To save segment storage space the forward index can now be [disabled](forward-index.md#disabling-the-forward-index) while creating new tables.
+When the encoding is `DICTIONARY`, the forward index does not physically store the actual row values but the dictionary ids.
```

Comment:
```suggestion
When the encoding is `DICTIONARY`, the forward index does not physically store the actual row values but the dictionary IDs.
```

Reply:
I've accepted all this ids -> IDs change, but TBH they look very strange to me. It is pretty common in technical documents to talk about `ids` instead of `IDs`. The later looks more like an ID card than the concept of `id` we use here.